daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -96,7 +96,7 @@ private[lf] object Pretty {
         text("Update failed due to a unknown contract") & prettyContractId(cid)
       case NonComparableValues =>
         text("functions are not comparable")
-      case ContractIdFreshness(globalCid) =>
+      case ContractIdComparability(globalCid) =>
```

Comment:
Did this sneak in from #10687?

Reply:
need more coffee.

Reply:
In fact, it is the contrary I sneak in  #10687 this. 

The correct place for this change is this PR, because this is related to V1 contract ID.